ui/src/lib/utils.ts

```diff
@@ -52,8 +52,10 @@ if (import.meta.vitest) {
       '11.06.2024, 13:07:45'
     );
 
-    expect(formatTimestamp('2024-06-11T13:07:45Z', 'UTC', 'fi-FI')).toBe(
-      '11.06.2024 klo 13.07.45'
+    // Test the Finnish locale with a custom format. Matching "klo" is made optional
+    // due to vitest isolated test environment failing to support the full ICU format.
+    expect(formatTimestamp('2024-06-11T13:07:45Z', 'UTC', 'fi-FI')).toMatch(
```

Comment:
I'd use [`toEqual`](https://vitest.dev/api/expect#toequal) instead to avoid the regexp.

Reply:
Oh it was actually needed. :D